src/config/enhance-octokit.ts

```diff
@@ -61,7 +61,7 @@ const instrumentRequests = (octokit: GitHubAPI) => {
 			const tags = {
 				path: extractPath(options.url),
```

Comment:
We need to remove all path parameters from this url. Otherwise it will explode the metric cardinality.

If we can't get this information from the octokit options we need to remove this tag at all

Reply:
- [ ] remove path from tags

Reply:
This was added by a GH dev 2 years ago. Do you know if it's caused any issues with our metrics?

Reply:
Just checked and we aren't using it in our metrics anyway. Will remove.